TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/sockets/BaseStringHandler.java

```diff
@@ -1,107 +1,33 @@
-                File dir = new File((String) data);
-                if (dir.exists() && dir.isDirectory()) FileDeleteStrategy.FORCE.deleteQuietly(dir);
-                break;
-            case TRANSFER_TEMPLATE:
-                try {
-                    InputStream inputStream = new ByteArrayInputStream(stringToByteArray((String) message.get("file")));
-                    switch ((String) message.get("transferType")) {
-                        case "SERVER_TEMPLATE":
-                            TimoCloudBase.getInstance().getTemplateManager().extractFiles(inputStream, new File(TimoCloudBase.getInstance().getFileManager().getServerTemplatesDirectory(), (String) message.get("template")));
-                            TimoCloudBase.getInstance().getSocketMessageManager().sendMessage(Message.create().setType(MessageType.SERVER_TRANSFER_FINISHED).setTarget(message.getTarget()));
-                            break;
-                        case "SERVER_GLOBAL_TEMPLATE":
-                            TimoCloudBase.getInstance().getTemplateManager().extractFiles(inputStream, TimoCloudBase.getInstance().getFileManager().getServerGlobalDirectory());
-                            TimoCloudBase.getInstance().getSocketMessageManager().sendMessage(Message.create().setType(MessageType.SERVER_TRANSFER_FINISHED).setTarget(message.getTarget()));
-                            break;
-                        case "PROXY_TEMPLATE":
-                            TimoCloudBase.getInstance().getTemplateManager().extractFiles(inputStream, new File(TimoCloudBase.getInstance().getFileManager().getProxyTemplatesDirectory(), (String) message.get("template")));
-                            TimoCloudBase.getInstance().getSocketMessageManager().sendMessage(Message.create().setType(MessageType.PROXY_TRANSFER_FINISHED).setTarget(message.getTarget()));
-                            break;
-                        case "PROXY_GLOBAL_TEMPLATE":
-                            TimoCloudBase.getInstance().getTemplateManager().extractFiles(inputStream, TimoCloudBase.getInstance().getFileManager().getProxyGlobalDirectory());
-                            TimoCloudBase.getInstance().getSocketMessageManager().sendMessage(Message.create().setType(MessageType.PROXY_TRANSFER_FINISHED).setTarget(message.getTarget()));
-                            break;
-                    }
-                    TimoCloudBase.getInstance().getInstanceManager().setDownloadingTemplate(false);
-                    break;
-                } catch (Exception e) {
-                    TimoCloudBase.getInstance().severe("Error while unpacking transferred files: ");
-                    TimoCloudBase.getInstance().severe(e);
-                }
-            default:
-                TimoCloudBase.getInstance().severe("Could not categorize json message: " + originalMessage);
-        }
+        getMessageHandlers(type).forEach(messageHandler -> messageHandler.execute(message, channel));
```

Comment:
We need some kind of exception handling so that the execution does not stop if one of the handlers throws an exception